platform/core/src/services/DicomMetadataStore/createStudyMetadata.js

```diff
@@ -13,6 +13,7 @@ function createStudyMetadata(StudyInstanceUID) {
      */
     addInstanceToSeries: function(instance) {
       const { SeriesInstanceUID } = instance;
+      if (!this.StudyDescription) this.StudyDescription = instance.StudyDescription;
```

Comment:
Recover the study description when teh first instance gets loaded - this instance is required to have the study description.

Reply:
use {} for the if statements, same as through out the code